exporter/awskinesisexporter/exporter.go

```diff
@@ -51,7 +51,16 @@ func createExporter(ctx context.Context, c component.ExporterConfig, log *zap.Lo
 	if !ok || conf == nil {
 		return nil, errors.New("incorrect config provided")
 	}
-	awsconf, err := awsconfig.LoadDefaultConfig(ctx)
+
+	var configOpts []func(*awsconfig.LoadOptions) error
+	if conf.AWS.Region != "" {
+		configOpts = append(configOpts, func(lo *awsconfig.LoadOptions) error {
+			lo.Region = conf.AWS.Region
+			return nil
+		})
+	}
```

Comment:
How is this change tested?

Reply:
I've been thinking about this, I don't think I have a good answer atm. 

Let me play with some ideas on how to expose this information.

Reply:
So how I wanted to do this and what came out are very different due to the fact the fact that using `assert.EqualValues`  resulted in differences due to lambda references being stored.

The best alternative without doing much of a major refactor was to create an options type and redefine the `kinesis.NewClient` method.